File: backend/src/main/java/com/example/course_project_2023/service/dto/ReviewDto.java
package com.example.course_project_2023.service.dto;

import jakarta.persistence.Column;

import java.util.Date;

public record ReviewDto(
        Long id,
        Short score,
        String nickname,
        String universityName,
        String subjectName,
        String reviewText,
        Date createdDate,
        Long likeCount,
        Long dislikeCount,
        String likeLink,
        String dislikeLink,
        Boolean isLiked,
        Boolean isDisliked
) {

}

File: frontend/src/app/models/ReviewDto.ts
export interface ReviewDto {
  id: number;
  score: number;
  nickname:string,
  universityName:string,
  subjectName: string;
  reviewText: string;
  createdDate: Date;
  likeCount: number;
  dislikeCount: number;
  likeLink: string;
  dislikeLink: string;
  isLiked: boolean;
  isDisliked: boolean;
}